iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -236,10 +236,9 @@ private static async Task<T> ReadAsAsync<T>(HttpContent content, CancellationTok
         {
             token.ThrowIfCancellationRequested();
 
+            var jsonSerializerOptions = new JsonSerializerOptions();
```

Comment:
Do we need options?

Reply:
Unfortunately, the method signature requires options to be passed in: https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserializeasync?view=net-7.0.
I can make it inline. 
(This is the http layer, so our client isn't using it directly today. If we decide to revert the removal of support in the future, we can make passing in this options configurable).